modules/powers/Nemesis.class.php

```diff
@@ -15,7 +15,148 @@ public function __construct($game, $playerId)
     $this->playerCount = [2, 3, 4];
     $this->golden  = false;
     $this->orderAid = 42;
+
+    $this->implemented = true;
+  }
+
+
+  public function stateAfterBuild()
+  {
+    $arg = [];
+    $this->argUsePower($arg);
+    return (count($arg['workers']) > 0)? 'power' : null;
+  }
+
+
+  public function argUsePower(&$arg)
+  {
+    $arg['power'] = $this->id;
+    $arg['power_name'] = $this->name;
+    $arg['skippable'] = true;
+    $arg['workers'] = $this->game->board->getPlacedActiveWorkers();
+    foreach($arg['workers'] as &$worker)
+    {
+      $worker['works'] = [];
+    }
+	$workers = $this->game->board->getPlacedActiveWorkers();
+	if (count($workers) != 2)
+	  throw new BgaVisibleSystemException('Unexpected state in Nemesis.');
+
+	$actions = $this->game->log->getLastWorks(['force'], null, 2);
```

Comment:
indeed, I thought no other force would happen in this turn, but playing e.g., Nemesis vs Harpies would not work here!

Reply:
I tried something like this but was not able to work it out. So I just banned Harpies and Charybdis to avoid opponent "force" during Nemesis turn. This requires a fix I did not find.